### ISSUE TITLE
Nick and Vanish 

### ISSUE THREAD

**Issue by D3viiid** [BedrockPlay/TODO-List#27]
Staff need commands /nick or /vanish. Hackers are smarter so when they see some staff rank they leave. We haven't any chance how to catch them. This commands will be very very helpful for staff.

**Comment by RajGamesYT** [comment]
I agree, its the most needed to catch hackers. Hackers leave when staff join. ;(. There after can't ban the hackers.
